### PR TITLE
fix: car example updated

### PR DESCRIPTION
**Pull request by lsmienk**
The car example needs to updated after the next release. This will affect the carList query that is used in this example. The data that we are removing now is still available in the car query.

### CODE REVIEW

File: car/cars.js [hunk 1]
```diff
@@ -8,24 +8,13 @@ export const displayCarsData = (cars = []) => {
   const profiles = [];
 
   cars.map(car => {
+    console.log(car.imagesData);
```
Comment (Khrystyna) [inline car/cars.js:11]: Let's remove console logs :)

**Review by Khrystyna — APPROVED** [review]

File: public/car/index.html [hunk 2]
```diff
@@ -25,29 +25,14 @@
         ></button>
       </div>
 
+      <div class="banner"><spec>Use the car ID in the car query to get more details about your EV</spec></div>
```
Comment (Khrystyna) [inline public/car/index.html:28]: Not really clear for me :) Perhps we could change the structure of the sentence a bit: To get more details about the car, please use `car(id: $carId)` query.

Comment (lsmienk) [reply public/car/index.html:28]: What about this? 
To get more details about about a car, please use the car query: car(id: $carId)

Comment (Khrystyna) [reply public/car/index.html:28]: Top! You can use `<pre>car(id: $carId)</pre>` to make a code  like text :)

**Pull request merged**